### ISSUE TITLE
Latex to ePub - Chinese characters causes bad display for tabular

### ISSUE THREAD

**Issue by jbardon** [jgm/pandoc#2348]
My Latex document contains some chinese characters (displayed with CJK package). Once I added these characters I noticed some troubles (with tabular environment) in the final display after having converted my .tex into an .epub file.

This sample code will make the bug appear:

```
\documentclass{book}
\usepackage{CJKutf8}

\newcommand{\hanyu}[1]{
    \begin{CJK}{UTF8}{gbsn}
        #1
    \end{CJK}
    \sbox0{\input{utf8.def}}
}

\title{Example}
\author{Jeremy}

\begin{document}
\chapter{First chapter}

\begin{tabular}{|c|l|}
  \hline

  \hanyu{你好} & hello\\
  \hanyu{再见} & good bye\\

  \hline
\end{tabular}

\end{document}
```

At the beginning I thought it was related to the new command which contains an environment (CJK) with multiple args. As a result I created the #2347 issue with a sample code which doesn't contains chinese characters. @nkalvi helped me to fix a first problem: my pandoc version.

To sum up: 
- Bad display of tabular with chinese characters (Latex -> ePub)
- Platform: OSX
- Pandoc version: 1.15.0.6

The command I use to compile (According to the [faq](http://pandoc.org/faqs.html):

```
$ pandoc Example2.tex -f latex -t epub -o Example.epub --latex-engine=xelatex -V mainfont='Adobe Ming Std'
```

**Comment by jgm** [comment]
Pandoc doesn't currently support the CJKutf8 package or its defined CJK environment.

Try replacing your definition of `hanyu` with:

    \newcommand{\hanyu}[1]{#1}

**Comment by jbardon** [comment]
I never thought about doing that because it doesn't compile with pdflatex or xelatex. However it works perfectly so pandoc is magic ! 

If my sources are good it's more about converting latex in html so no need to compile the latex but ok it just seems not logical for me (to convert not working tex code).

**Comment by jgm** [comment]
+++ Jérémy Bardon [Aug 06 15 09:58 ]:
>   I never thought about doing that because it doesn't compile with
>   pdflatex or xelatex. However it works perfectly so pandoc is magic !
>
>   If my sources are good it's more about converting latex in html so no
>   need to compile the latex but ok it just seems not logical for me (to
>   convert not working tex code).

It's just a workaround, to compensate for the fact that
pandoc can't support every conceivable latex package.
(Though I don't see why this wouldn't work with xelatex, if
you've got the right fonts; it's just literal unicode
characters.)

**Comment by jbardon** [comment]
I totally agree with you because it surprised me when I tried to compile my latex file (in utf-8) and it didn't worked before adding the right package.

As I said before it's the sample is a part of my document and here is a problem with the tabbing environment (just replace the tabular from the original sample):

```
\begin{tabbing}
\hspace{2cm} \= \hspace{2cm} \= \hspace{2cm} \= \hspace{2cm} \= \hspace{2cm} \= \kill
\hanyu{一} \> \hanyu{二} \> \hanyu{三} \> \hanyu{四} \> \hanyu{五} \\
\hanyu{六} \> \hanyu{七} \> \hanyu{八} \> \hanyu{九} \> \hanyu{十}\\
\end{tabbing}
```

The output includes equals characters, I'm not a Latex expert so maybe my way to do tabbing is not right
![capture d ecran 2015-08-06 a 19 30 07](https://cloud.githubusercontent.com/assets/9324783/9118574/99b049be-3c71-11e5-9656-bf2cb05c6493.png)

**Comment by jgm** [comment]
The tabbing environment isn't yet supported. If you want
support for it, please open a separate issue requesting
this.

+++ Jérémy Bardon [Aug 06 15 10:31 ]:

**Comment by jbardon** [comment]
So I think you can now close this issue. I created another one to request the tabbing support #2349. 
Thank's for your help!
